src/test/java/tests/LoginTest.java

```diff
@@ -0,0 +1,78 @@
+package tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.MainPage;
+import pages.RegistrationPage;
+import properties.Property;
+import util.RandomSymbolUtil;
+
+
+public class LoginTest extends BaseTest {
+
+    @Test(description = "Positive test to check if the entrance form opens")
+    public void entranceFormTest() {
+        MainPage newMainPage = new MainPage()
+                .openPage()
+                .openEntranceForm();
+        Assert.assertEquals(newMainPage.getLoginViaNickNameMessage(), "или через ник, e-mail", "Message under login field wasn't found.");
+    }
+
+    @Test(description = "Positive test for registration of a new user", enabled = false)
+    public void registrationTest() {
+        RegistrationPage registrationPage = new MainPage()
+                .openPage()
+                .openEntranceForm()
+                .goToRegistrationLink()
+                .setLogin(RandomSymbolUtil.getRandomLogin())
+                .setPassword(RandomSymbolUtil.getRandomPassword())
+                .setRepeatPassword(RandomSymbolUtil.getRandomPassword())
+                .acceptPrivacyPolicy()
+                .clickRegistrationButton();
+        Assert.assertEquals(registrationPage.emailConfirmationMessage(), "Подтвердите ваш e-mail", "Fail to sign up.");
+    }
+
+    @Test(description = "Positive test for login with correct credentials")
+    public void logInTest() {
+        MainPage mainPage = new MainPage()
+                .openPage()
+                .openEntranceForm()
+                .insertLogin(props.getKeyProperty(Property.LOGIN))
```

Comment:
try to hide props.getKeyProperty(Property.LOGIN) from the test, think of the ways how to do that

Reply:
done